### PR TITLE
Updated change log and version numbers.

### PR DESCRIPTION
**Pull request by vedanshujain**
This PR currently target master. We will cut a release branch from the master branch after this PR is merged.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -2,9 +2,9 @@
 Contributors: automattic, mikejolley, jameskoster, claudiosanches, kloon, rodrigosprimo, peterfabian1000, vedjain, jamosova, obliviousharmony
 Tags: e-commerce, store, sales, sell, woo, shop, cart, checkout, downloadable, downloads, payments, paypal, storefront, stripe, woo commerce
 Requires at least: 5.0
-Tested up to: 5.3
+Tested up to: 5.4
 Requires PHP: 7.0
-Stable tag: 4.0.0
+Stable tag: 4.1.0
```
Comment (peterfabian) [inline readme.txt:7]: Shouldn't this point to 4.0.1?

**Review by peterfabian — CHANGES_REQUESTED** [review]
I think the stable release should not change, otherwise I think it's fine.

File: readme.txt (hunk 1 above)
Comment (vedanshujain) [reply readme.txt]: ah right. updated in https://github.com/woocommerce/woocommerce/pull/26097/commits/8e2a6b2c8233fca13b16814080f94408aad9d029.

**Review by peterfabian — APPROVED** [review]
Looks good now.

**Pull request merged**